### ISSUE TITLE
Force Respawn

### ISSUE THREAD

**Issue by Mako88** [dxx-rebirth/dxx-rebirth#107]
Add a host-settable option to force respawns (after 5 seconds, or something like that). This would be to prevent people waiting extended periods of time before respawning (which gets quite annoying in 1v1 games.)

This is a result of a discussion I had with a couple of guys in the community who requested this feature.

NOTE: This would still allow people to respawn as quickly as they'd like, it'd just force a respawn after 5 seconds.

**Comment by vLKp** [comment]
This seems like it could cause a new problem.  If a player force-respawns while he is away from his computer, he will be easy prey, and then force-respawn again soon after that.  You go from the frustration of the living player unable to face an enemy to instead the frustration of the dead player that he is repeatedly killed (affecting rank, kill goals, etc.) while he is not even playing.

**Comment by Mako88** [comment]
That's true...however I don't think it would really be an issue necessarily. The only games which would need this option would be fairly serious ones (since in non-serious games it doesn't really matter if you don't respawn for awhile), and in serious games if someone goes afk, they either call a timeout first, or forfeit the match anyway.
